From ca2eedaad0bc27dd6a540e61c583f0b745641a84 Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Tue, 2 Dec 2014 16:39:28 +0200
Subject: [PATCH 1/2] man: Remove client attribute description

Remove attribute description from clients.conf(5) source as it is
outdated, lists just a few attributes and wasn't updated for more than a
year. Refer to clients.conf file itself, instead.
---
 man/man5/clients.conf.5 | 74 +++----------------------------------------------
 1 file changed, 4 insertions(+), 70 deletions(-)

diff --git a/man/man5/clients.conf.5 b/man/man5/clients.conf.5
index 9af246a..f9207d1 100644
--- a/man/man5/clients.conf.5
+++ b/man/man5/clients.conf.5
@@ -26,76 +26,10 @@ client <short-name> {
        <attribute> = <value>
  }
 .fi
-.SH ATTRIBUTES
-The attributes that can appear in a 
-.B client
-section are listed below. Required attributes are labelled as
-such. All other attributes are optional.
-.TP 0.5i
-.B ipaddr [Required]
-The IP address of the client.  For IPv6, use "ipv6addr"
-.TP 0.5i
-.B secret [Required]
-The RADIUS shared secret used for communication between the client/NAS
-and the RADIUS server.
-.TP 0.5i
-.B shortname [optional]
-A short alias that can be used in place of the IP address or fully
-qualified hostname provided in the first line of the section.
-.TP 0.5i
-.B nas_type
-The nas_type attribute is used to tell the 
-.BR checkrad.pl
-script which NAS-specific method it should use when checking
-simultaneous use.
-
-The following values are currently recognized:
-.nf
-cisco
-computone
-livingston
-max40xx
-multitech
-netserver
-pathras
-patton
-portslave
-tc
-usrhiper
-other
-.fi
-.TP 0.5i
-.B login
-Reserved for future use.
-.TP 0.5i
-.B password
-Reserved for future use.
-.SH EXAMPLES
-.IP
-.nf
-client localhost {
-        ipaddr          = 127.0.0.1
-        secret          = testing123
-        shortname       = localhost
-        nas_type         = other     
-}
-.fi
-.LP
-This adds a client for the loopback address. This is useful in testing
-the 
-server locally, for example with 
-.BR radclient (1).
-.IP
-.nf
-client private-network-1 {
-        ipaddr          = 192.0.2.0
-        netmask         = 24
-        secret          = testing123-1
-        shortname       = private-network-1
-}
-.fi
-.LP
-This entry represents any client from the 192.0.2.0/24 network. 
+.PP
+Clients have many configuration parameters. Most are documented in the file
+itself as comments. This page documents only the format of the file. Please
+read the \fBclients.conf\fP file itself for more information.
 
 The old-style format from 1.x is still accepted by the server, but
 that form is deprecated.
-- 
2.1.3

